Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -226,10 +228,20 @@ class RubricCircleView: UIView {
         let howManyCanFitInWidth = CGFloat( ceil( maxWidth / (w + space) ) )
         if howManyCanFitInWidth == 0 { return 0 }
         let rows = CGFloat(ceil(count / howManyCanFitInWidth))
+        //  If both hideRubricPoints and freeFormCriterionComments are set,
+        //  circle view should not be visible. Only comments should show of rubric.
+        if rubric.hideRubricPoints && rubric.freeFormCriterionComments { return 0 }
```

Comment:
Looks like you check these two properties together a few different place. Do you think a getter on the rubric model would be beneficial? 

```swift
var onlyShowComments: { hideRubricPoints && freeFormCriterionComments }
```
Maybe?